controllers/comments-controllers.js

```diff
@@ -18,3 +21,35 @@ exports.getCommentsByArticleId = (request, response, next) => {
       next(err);
     });
 };
+
+exports.postCommentByArticleId = (request, response, next) => {
+  const { article_id } = request.params;
+  const username = request.body.username;
+  const newComment = request.body.comment;
+
+  if (username === undefined || newComment == undefined) {
+    return Promise.reject({
+      status: 400,
+      msg: "Comment must have both username and body properties",
+    }).catch((err) => {
+      next(err);
+    });
```

Comment:
What is this catch block acheiving here? Would a do we need to catch errors from a  returned `Promise.reject`?
I'd recommend moving these two `Promise.reject`s to your model. Then they will be caught by your `catch` block on line 52, and make your code much DRYer.